### ISSUE TITLE
Accidental mistake of double sigmoid operations?

### ISSUE THREAD

**Issue by jianhaoooooo** [hunglc007/tensorflow-yolov4-tflite#157]
In `decode_train`, line 189-190
```
    pred_conf = tf.sigmoid(conv_raw_conf)
    pred_prob = tf.sigmoid(conv_raw_prob)
```

This then forms part of the model outputs and when it comes to `compute_loss`, at line 350-356

```
    conf_loss = conf_focal * (
            respond_bbox * tf.nn.sigmoid_cross_entropy_with_logits(labels=respond_bbox, logits=conv_raw_conf)
            +
            respond_bgd * tf.nn.sigmoid_cross_entropy_with_logits(labels=respond_bbox, logits=conv_raw_conf)
    )

    prob_loss = respond_bbox * tf.nn.sigmoid_cross_entropy_with_logits(labels=label_prob, logits=conv_raw_prob)

```

However, `conv_raw_conf` and `conv_raw_prob` are actually `pred_conf` and `pred_prob` in this case. Since `tf.nn.sigmoid_cross_entropy_with_logits`  computes sigmoid for its logits input, it seems to me that this ends up as a repeated double sigmoid operations for `conv_raw_conf` and `conv_raw_prob`?

If this is indeed intentional, I would greatly appreciate if anyone could help explaining why.

Thanks!
